### DISCUSSION TITLE
Please deliver RSS feeds for thematic categories (topics)

### DISCUSSION THREAD

**Discussion by jbionic2016** [community/community#61463] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi there,

GitHub is truly great in many aspects. However I couldn't find any way to retrieve RSS updates for the list of topics of my interest, such as, for instance , Chrome extensions or OSINT tools (see below links), to which I would love to subscribe:
https://github.com/topics/chrome-extension?o=desc&s=updated
https://github.com/topics/osint?o=desc&s=updated

I suppose the feature must be available freely to everyone because this helps to improve the statistics of site visits and downloads.
Deployment of RSS is free and easy to deliver, so I'd be happy to advise you on how to do that:

Creating RSS feeds would seem very easy and technically straightforward once you watched the following video 
https://www.youtube.com/watch?v=PTXnmMlipmo
especially pay attention to the part that starts at 3:40 about the RSS builder.
The solution is completely free and easy to manage.

With best wishes.

**Comment by leafstrat** [comment, marked as answer]
https://rsshub.app/github/topics/osint/o=desc&s=updated


details:
https://docs.rsshub.app/en/programming.html#github
